File: src/Meatproducing.cpp
#include <iostream>
#include <string>
#include "FarmAnimal.h"
#include "Meatproducing.h"

using namespace std;

int Meatproducing::jlhMeatP =0;

Meatproducing::Meatproducing(int x,int y) : FarmAnimal(x,y){
    jlhMeatP++;
}

Meatproducing::~Meatproducing(){
    //cout<<"One of your meat producing animal has died"<<endl;
}

int Meatproducing::getjlhMeatP(){
    return jlhMeatP;
}

File: java/src/LivingThings/Animals/MeatProducing.java
public class MeatProducing {
    private static int jlhMeatP =0;

    public MeatProducing(int x,int y){
        //super(x,y);
        MeatProducing.jlhMeatP++;
    }
    public static Integer getjlhMeatP() {
        return jlhMeatP;
    }
}